### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by Nootaku** [Deluze/electron-vue-template#29] [General]
Hello,

I'm Max (from Belgium). A while back I created a pure Electron App under the GNU GENERAL PUBLIC LICENSE (v3). I wanted to revamp it and stumbled upon your template that looked like a good start for my project.

I was actually wondering how the MIT licensing was working. Is it possible to have both licenses co-exist ? (MIT for the template, GPL for the UI / backend).

Could you please advise as I definitely want to credit your work if I use your template but the first version of my project is already under another license.

Kind regards

**Comment by Deluze** [comment]
Multi licensing is definitely a thing. As per MIT you're allowed to modify all of the code and do whatever you want with it. As long as you keep a copy of the MIT license somewhere in your project you are good.
